Doc/library/zipfile.rst

```diff
@@ -289,7 +289,7 @@ ZipFile Objects
       compressed text files in :term:`universal newlines` mode.
 
    .. versionchanged:: 3.6
-      :meth:`open` can now be used to write files into the archive with the
+      :meth:`~ZipFile.open` can now be used to write files into the archive with the
```

Comment:
Not sure that I would hide `ZipFile` here; being explicit would benefit readers.

Reply:
```suggestion
      :meth:`ZipFile.open` can now be used to write files into the archive with the
```